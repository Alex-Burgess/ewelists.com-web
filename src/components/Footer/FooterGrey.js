import React from "react";
import { Link } from "react-router-dom";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
// core components
import Footer from "components/Footer/Footer.js";

import styles from "assets/jss/material-kit-pro-react/components/footer/footerStyle.js";
const useStyles = makeStyles(styles);

export default function FooterGrey(props) {
  const classes = useStyles();

  return (
    <div>
      <div className={classes.spacer} />
      <Footer
        theme="grey"
        content={
          <div>
            <div className={classes.left}>
              <List className={classes.list}>
                <ListItem className={classes.inlineBlock}>
                  <Link to="/privacy">Privacy</Link>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <Link to="/terms">Terms</Link>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <Link to="/contact">Support</Link>
                </ListItem>
              </List>
            </div>
            <div className={classes.rightLinks}>
              <a href="/" className={classes.block}>
                &copy; {1900 + new Date().getYear()} ewelists
              </a>
            </div>
          </div>
        }
      />
    </div>
  );
}
