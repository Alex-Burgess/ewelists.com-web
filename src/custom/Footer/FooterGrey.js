import React from "react";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
// core components
import Footer from "components/Footer/Footer.js";

import styles from "assets/jss/custom/components/footerGreyStyle.js";
const useStyles = makeStyles(styles);

export default function FooterGrey(props) {
  const classes = useStyles();

  return (
    <div>
      <div className={classes.spacer} />
      <Footer
        className={classes.footer}
        content={
          <div>
            <div className={classes.left}>
              <List className={classes.list}>
                <ListItem className={classes.inlineBlock}>
                  <a href="/privacy" className={classes.block}>
                    Privacy
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a href="/terms" className={classes.block}>
                    Terms
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a href="/contact"  className={classes.block}>
                    Support
                  </a>
                </ListItem>
              </List>
            </div>
            <div className={classes.rightLinks}>
              <a href="/" className={classes.block}>
                &copy; {1900 + new Date().getYear()} ewelists
              </a>
            </div>
          </div>
        }
      />
    </div>
  );
}
