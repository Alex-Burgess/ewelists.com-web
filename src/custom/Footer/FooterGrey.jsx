/*!

=========================================================
* Material Kit PRO React - v1.7.0
=========================================================

* Product Page: https://www.creative-tim.com/product/material-kit-pro-react
* Copyright 2019 Creative Tim (https://www.creative-tim.com)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/
/*eslint-disable*/
import React from "react";
// nodejs library to set properties for components
import PropTypes from "prop-types";
// @material-ui/core components
import withStyles from "@material-ui/core/styles/withStyles";
import InputAdornment from "@material-ui/core/InputAdornment";
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
import Icon from "@material-ui/core/Icon";
// @material-ui/icons
import Face from "@material-ui/icons/Face";
// core components
import Footer from "components/Footer/Footer.jsx";
import Button from "components/CustomButtons/Button.jsx";

import footerDarkStyle from "assets/jss/custom/components/footerGreyStyle.jsx";

function FooterDark(props) {
  const { classes } = props;

  return (
    <div>
      <div className={classes.spacer} />
      <Footer
        className={classes.footer}
        content={
          <div>
            <div className={classes.left}>
              <List className={classes.list}>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/"
                    className={classes.block}
                  >
                    &copy; {1900 + new Date().getYear()} ewelists
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/contact"
                    className={classes.block}
                  >
                    Contact Us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/#about"
                    className={classes.block}
                  >
                    About us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/privacy"
                    className={classes.block}
                  >
                    Privacy
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/terms"
                    className={classes.block}
                  >
                    Terms
                  </a>
                </ListItem>
              </List>
            </div>
            <div className={classes.rightLinks}>
              <ul>
                <li>
                  <Button
                    href="https://www.facebook.com/ewelists"
                    target="_blank"
                    color="facebook"
                    justIcon
                    simple
                  >
                    <i className="fab fa-facebook" />
                  </Button>
                </li>
                <li>
                  <Button
                    href="https://twitter.com/ewelists"
                    target="_blank"
                    color="twitter"
                    justIcon
                    simple
                  >
                    <i className="fab fa-twitter" />
                  </Button>
                </li>
                <li>
                  <Button
                    href="https://www.instagram.com/ewelists/"
                    target="_blank"
                    color="instagram"
                    justIcon
                    simple
                  >
                    <i className="fab fa-instagram" />
                  </Button>
                </li>
              </ul>
            </div>
          </div>
        }
      />
    </div>
  );
}

FooterDark.propTypes = {
  classes: PropTypes.object
};

export default withStyles(footerDarkStyle)(FooterDark);
