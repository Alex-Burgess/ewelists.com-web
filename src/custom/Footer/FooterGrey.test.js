import * as React from 'react';
import renderer from 'react-test-renderer';
import FooterGrey from './FooterGrey';

test('Renders FooterGrey component.', () => {
  const tree = renderer
    .create(
      <FooterGrey />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
