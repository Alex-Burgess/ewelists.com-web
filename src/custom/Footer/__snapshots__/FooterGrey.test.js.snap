// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders FooterGrey component. 1`] = `
<div>
  <div />
  <footer
    className="makeStyles-footer-12 makeStyles-footer-7"
  >
    <div
      className="makeStyles-container-18"
    >
       
      <div>
        <div
          className="makeStyles-left-4"
        >
          <ul
            className="MuiList-root makeStyles-list-3 MuiList-padding"
          >
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/"
              >
                Â© 
                2020
                 ewelists
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/contact"
              >
                Contact Us
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/#about"
              >
                About us
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/privacy"
              >
                Privacy
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/terms"
              >
                Terms
              </a>
            </li>
          </ul>
        </div>
        <div
          className="makeStyles-rightLinks-8"
        >
          <ul>
            <li>
              <a
                aria-disabled={false}
                className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-44 makeStyles-facebook-55 makeStyles-simple-66 makeStyles-justIcon-74"
                href="https://www.facebook.com/ewelists"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                target="_blank"
              >
                <span
                  className="MuiButton-label"
                >
                  <i
                    className="fab fa-facebook"
                  />
                </span>
              </a>
            </li>
            <li>
              <a
                aria-disabled={false}
                className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-44 makeStyles-twitter-54 makeStyles-simple-66 makeStyles-justIcon-74"
                href="https://twitter.com/ewelists"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                target="_blank"
              >
                <span
                  className="MuiButton-label"
                >
                  <i
                    className="fab fa-twitter"
                  />
                </span>
              </a>
            </li>
            <li>
              <a
                aria-disabled={false}
                className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-44 makeStyles-instagram-65 makeStyles-simple-66 makeStyles-justIcon-74"
                href="https://www.instagram.com/ewelists/"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                target="_blank"
              >
                <span
                  className="MuiButton-label"
                >
                  <i
                    className="fab fa-instagram"
                  />
                </span>
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="makeStyles-clearFix-28"
      />
    </div>
  </footer>
</div>
`;
